README.md

```diff
@@ -1,10 +1,20 @@
-# Nested Tags And Attributes
+# Nested HTML Tags And Attributes
 
-Explanation of content.
+When you take a look at an entire HTML document, it's sometimes challenging to
+keep track of where tags begin and end. But understanding more about HTML tag
+nesting and attributes can help everything fall into place.
 
-### Nested Tags
+## Students will be able to ...
 
-Tags can be nested as demonstrated in the navigation snippet below.
+1. Assess HTML document structure
+2. Identify nested HTML tags
+3. Recognize HTML tag attributes and their purposes
+
+## Nested Tags
```

Comment:
H2 should be objectives (like, literally copy/paste the objective). Maybe make this one an H3?